packages/devtools-launchpad/src/components/Root.css

```diff
@@ -1,6 +1,18 @@
 :root.theme-light,
 :root .theme-light {
   --theme-search-overlays-semitransparent: rgba(221, 225, 228, 0.66);
+  --theme-codemirror-gutter-background: #f4f4f4;
+}
+
+:root.theme-dark,
+:root .theme-dark {
+  --theme-search-overlays-semitransparent: rgba(42, 46, 56, 0.66);
+  --theme-codemirror-gutter-background: #262b37;
+}
+
+:root.theme-firebug,
+:root .theme-firebug {
+  --theme-codemirror-gutter-background: #ebeced;
```

Comment:
why not move them to the debugger project? I'd like to see them in a variables.css file that's pulled in `App.js`